### PR TITLE
[VMD] Source was not properly packaged in the viewmodels-declarative-compose artefact

### PR DESCRIPTION
**Pull request by gbourassa**
## Description

Since source was not properly packaged, it was not possible to browser the code of all the VMD binding for jetpack compose, when used in a project.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: trikot-viewmodels-declarative/compose/build.gradle.kts [hunk 1]
```diff
@@ -27,6 +28,9 @@ android {
     kotlinOptions {
         jvmTarget = "11"
     }
+    sourceSets {
+        println("SSS $this")
```
Comment (marcantoinefortier) [inline trikot-viewmodels-declarative/compose/build.gradle.kts:32]: 👀

Comment (gbourassa) [reply trikot-viewmodels-declarative/compose/build.gradle.kts:32]: 🤦

**Review by marcantoinefortier — APPROVED** [review]
LGTM 👀

**Review by mathieularue — APPROVED** [review]
🎉

File: trikot-viewmodels-declarative/compose/build.gradle.kts [hunk 2]
```diff
@@ -50,25 +51,26 @@ dependencies {
...
         archiveClassifier.set("sources")
-        from(android.sourceSets.getByName("main").java.srcDirs().toString())
+    }
+
+    artifacts {
+        archives(sourcesJar)
     }
 }
 
+
```
Comment (remi) [inline trikot-viewmodels-declarative/compose/build.gradle.kts:65]: Is this wanted? There doesn’t seem to be other double blank lines anywhere else in the file.

**Pull request merged**